index.d.ts

```diff
@@ -562,7 +562,7 @@ export class Pushgateway {
 	pushAdd(
 		params: Pushgateway.Parameters,
 		callback: (error?: Error, httpResponse?: any, body?: any) => void,
-	): void;
+	): Promise<{ resp: any, body: any }>;
```

Comment:
```suggestion
	): Promise<{ httpResponse?: any, body?: any }>;
```

just to match what we have.

We should also probably type them as `unknown` and not `any`

Reply:
it seems like I cannot set resp to httpResponse - we'd have to change here too https://github.com/siimon/prom-client/blob/f177b1fd3d4db5fc48fcb1ec02d94069fffcf144/lib/pushgateway.js#L71